shvspy/src/dlgmountseditor.h

```diff
@@ -27,18 +28,31 @@ class DlgMountsEditor : public QDialog
 
 	QString selectedMount();
 	void listMounts();
+	void getMountPoint(const QString &id);
+	void getMountDescription(const QString &id);
 	//void callDeleteAccessForMount(const std::string &mount);
 
 	void onAddMountClicked();
 	void onDeleteMountClicked();
 	void onEditMountClicked();
 	void onTableMountDoubleClicked(QModelIndex ix);
+	void onRpcCallsFinished();
+	void setFilter(const QString &filter);
 
 	void setStatusText(const QString &txt);
 private:
+	struct MountPointInfo {
+		QString id;
+		QString mountPoint;
+		QString description;
+	};
 	Ui::DlgMountsEditor *ui;
 	std::string m_aclEtcNodePath;
 	shv::iotqt::rpc::ClientConnection *m_rpcConnection = nullptr;
+	QMap<QString, MountPointInfo> m_mountPoints;
+	bool m_rpcCallFailed;
+	int m_rpcCallsToComplete;
```

Comment:
ja tyhle promenny pro stav nejaky funkce v class fieldech nemam rad, proc proste neopalis ty RPC cally a nevyplnujes tabulku, jak to dochazi, nemusis hlidat, kolik jich prislo.